package.json

```diff
@@ -1,115 +1,125 @@
 {
-  "name": "streamline",
-  "version": "0.0.0",
-  "license": "MIT",
+  "name": "@getluko/streamline",
+  "version": "0.0.53",
   "scripts": {
-    "prepare": "husky install",
-    "postinstall": "patch-package",
-    "sandbox:start": "yarn nx run sandbox:start",
-    "sandbox:start:e2e": "EXPO_PUBLIC_E2E=true yarn nx run sandbox:start-e2e",
-    "sandbox:start:clear": "yarn nx run sandbox:start --clear",
-    "sandbox:test": "yarn nx run streamline:test",
-    "sandbox:test:visual:ios": "yarn nx run sandbox:test-visual-ios",
-    "sandbox:test:visual:ios:compare": "yarn nx run sandbox:test-visual-ios-compare",
-    "sandbox:test:visual:android": "yarn nx run sandbox:test-visual-android",
-    "sandbox:test:visual:android:compare": "yarn nx run sandbox:test-visual-android-compare",
```

Comment:
What about all these scripts ?

Reply:
you can do `yarn sandbox test:visual:ios` instead